Cargo.toml

```diff
@@ -11,6 +11,12 @@ members = [
     "stacks-signer",
     "testnet/stacks-node"]
 
+# Dependencies we want to keep the same between workspace members
+[workspace.dependencies]  
+wsts = { version = "7.0", default-features = false }
```

Comment:
This is the correct way to include `wsts` to avoid including `bindgen` and friends.